core_backend/tests/api/test_manage_content.py

```diff
@@ -96,7 +96,7 @@ def test_edit_and_retrieve_content(
         content_title: str,
         content_text: str,
         fullaccess_token: str,
-        readonly_token: str,
+        # readonly_token: str,
```

Comment:
Same comment as other PR. Let's think of test cases since this is getting complex.